addons/website/static/src/js/content/menu.js

```diff
@@ -659,6 +660,13 @@ publicWidget.registry.HeaderMainCollapse = publicWidget.Widget.extend({
                     this.options.wysiwyg && this.options.wysiwyg.odooEditor.observerActive();
                 }
             }
+            // Specific case with the "boxed" header template where the "call to
+            // action" button is inaccessible in the "off-canvas" mobile menu.
+            this.offcanvasAndBoxedHeader = false;
+            if (weUtils.getCSSVariableValue('header-template').includes('boxed')) {
+                this.boxedHeaderCallToActionEl = this.$target[0].querySelector('#top_menu_container .oe_structure_solo');
```

Comment:
Since we don't know how a user could have customized the header, I think that the selector should be a bit more precise, to be sure to target the one that doesn't go in the mobile menu. Maybe `#top_menu_collapse` instead of `#top_menu_container` ? :) (just to be sure to not break anything in stable)

Reply:
I made the changes :+1: